WooCommerce/Classes/System/WooConstants.swift

```diff
@@ -155,8 +155,11 @@ extension WooConstants {
         ///
 #if DEBUG
         case inAppFeedback = "https://automattic.survey.fm/woo-app-general-feedback-test-survey"
+        case IPPinAppFeedback = "https://automattic.survey.fm/woo-app-ipp-in-app-feedback-testing"
```

Comment:
Now that we have two inApp feedbacks, the naming looks a bit confusing to me. To make it clearer I would simply name it 
`generalFeedback` and
`IPPFeedback`
We can omit the `inApp` prefix, we know that the feedback is triggered from the app.

Reply:
Definitely, changed these at https://github.com/woocommerce/woocommerce-ios/pull/8532/commits/34b894e3754b7f303e0911501f4a70a39c1ea54f and https://github.com/woocommerce/woocommerce-ios/pull/8532/commits/7a13d16506c96f3791ab6e0735c61b5998c7e56e